apps/cowswap-frontend/src/modules/twap/hooks/useAllEmulatedOrders.ts

```diff
@@ -17,16 +18,14 @@ export function useAllEmulatedOrders(): Order[] {
   const emulatedPartOrders = useEmulatedPartOrders(twapOrdersTokens)
   const isSafeApp = useIsSafeApp()
 
-  const limitOrders = useOrders(chainId, account, OrderClass.LIMIT)
+  const limitOrders = useOrders(chainId, account, UiOrderType.TWAP)
```

Comment:
Name of the const `limitOrders`, but we use `UiOrderType.TWAP`. Something doesn't match

Reply:
Good catch!
Before this change, we were filtering all limit orders and then later we try to find out whether they are twap or not.
This change makes the query narrower.
I'll update the constant name to reflect that.